modules/op-triggering-a-pipelinerun.adoc

```diff
@@ -7,20 +7,20 @@
 
 Whenever a `push` event occurs in the Git repository, the configured Webhook sends an event payload to the publicly exposed EventListener service route. The EventListener service of the application processes the payload, and passes it to the relevant TriggerBindings and TriggerTemplates pair. The TriggerBinding extracts the parameters and the TriggerTemplate uses these parameters to create resources. This may rebuild and redeploy the application.
 
-In this section, you will push an empty commit to the front-end `vote-api` repository, which will trigger the PipelineRun.
+In this section, you will push an empty commit to the front-end `vote-ui` repository, which will trigger the PipelineRun.
```

Comment:
I know this assembly uses the convention of, "In this section, you will..." but it doesn't seem to align with our overall docs practice of using a more active voice (as opposed to "you will").

If you're so inclined, I'd suggest revising to something like, 

> The following example describes how to push an empty commit to the front-end `vote-ui` repository, which triggers the PipelineRun.